docs/config/index.md

```diff
@@ -524,6 +524,20 @@ createServer()
   }
   ```
 
+### server.publicPath
+
+- **Type:** `string`
+
+  This option allows the user to customize the URL prefix. Example:
+
+  ```js
+  export default {
+    server: {
+      publicPath: 'http://127.0.0.1:8080/'
+    }
+  }
```

Comment:
We try to use `defineConfig` everywhere in the docs, so folks know that this is a thing and prefer to use it

```suggestion
  export default defineConfig({
    server: {
      publicPath: 'http://127.0.0.1:8080/'
    }
  })
```